### ISSUE TITLE
unexpected print out to console when using saveWorkbook

### ISSUE THREAD

**Issue by AdrianHordyk** [awalker89/openxlsx#378]
## Expected Behavior

Expecting nothing printed to console when calling `openxlsx::saveWorkbook`

## Actual Behavior
A bunch of text printed to console:

```
  adding: [Content_Types].xml (deflated 72%)
  adding: _rels/ (stored 0%)
  adding: _rels/.rels (deflated 60%)
  adding: docProps/ (stored 0%)
  adding: docProps/app.xml (deflated 39%)
  adding: docProps/core.xml (deflated 47%)
  adding: xl/ (stored 0%)
  adding: xl/drawings/ (stored 0%)
  adding: xl/printerSettings/ (stored 0%)
  adding: xl/styles.xml (deflated 54%)
  adding: xl/tables/_rels/ (stored 0%)
  ...
  ...
```

## Steps to Reproduce the Problem 

```
library(openxlsx)
wb <- openxlsx::createWorkbook()
openxlsx::saveWorkbook(wb, file="example.xlsx")
```

## sessionInfo()

```
R version 3.5.0 (2018-04-23)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 16299)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] openxlsx_4.0.17

loaded via a namespace (and not attached):
[1] compiler_3.5.0 Rcpp_0.12.16  
```

  - Version of openxlsx: 4.0.17
  - Version of R: R version 3.5.0 (2018-04-23)

**Comment by philbastian** [comment]
I'm having the same problem ever since upgrading to R version 3.5.0. I've tried uninstalling and reinstalling both R and Rtools (apparently this really annoys me) in addition to openxlsx. I also tried to install using devtools, but I was getting a different error with that.

**Comment by awalker89** [comment]
I believe this has been resolved in the current CRAN version 4.1.0. Please re-open if still an issue.
